journal/original/16-chapter-13.adoc

```diff
@@ -1188,6 +1229,9 @@ The return of the sheriff of Lancaster was thus:--
 [.small-break]
 '''
 
+[.embedded-content-document.letter]
```

Comment:
Change this one to `legal` too. 👮‍♀️  And, let's nix the small-break above here as well.